types/recordrtc/index.d.ts

```diff
@@ -54,7 +54,7 @@ declare namespace RecordRTC {
         frameInterval?: number;
 
         /** if you are recording multiple streams into single file, this helps you see what is being recorded */
-        previewStream?(cb: (stream: MediaStream) => void): void;
+        previewStream?: (stream: MediaStream) => void;
```

Comment:
oops, these three (`previewStream`, `onTimeStamp`, `ondataavailable`) were wrong initially. Not sure why I thought they were callbacks. This new version is correct.